### DISCUSSION TITLE
[blocks]: Example Tasks

### DISCUSSION THREAD

**Discussion by dEvAshirvad** [shadcn-ui/ui#3356] [Blocks Request]
### Description

I am fascinated with that data table but same example is not in docs it's simple. It will be big help if we can recreate those

### Example

https://ui.shadcn.com/examples/tasks

**Comment by Abdulkareemoj** [comment]
https://github.com/shadcn-ui/ui/tree/main/apps/www/app/(app)/examples/tasks
This one?

**Comment by imfarhat** [comment]
https://github.com/shadcn-ui/ui/tree/main/deprecated/www/app/(app)/examples/tasks here you go
